apps/mobile/src/features/Onboarding/components/OnboardingCarousel/items.tsx

```diff
@@ -17,6 +18,9 @@ const styles = StyleSheet.create({
   signing: {
     height: Math.abs(windowHeight * 0.3),
   },
+  notifications: {
+    height: Math.abs(windowHeight * 0.32),
```

Comment:
what's up with those magic contants here? For signing it is 0.3, here it is 0.32? Why are we multiplying with that? How does it look on a smaller/bigger device?